EIPS/eip-7702.md

```diff
@@ -88,11 +151,33 @@ Specifically:
 
 ## Backwards Compatibility
 
-This EIP breaks the invariant that an account balance can only decrease as a result of transactions originating from that account. This has consequences for mempool design, and for other EIPs such as inclusion lists. However, these issues are common to any proposal that provides similar functionality, including EIP-3074.
+This EIP breaks the invariant that an account balance can only decrease as a result of transactions originating from that account. It also breaks the invariant that an EOA nonce may not increase after transaction execution has begun. These breakages have consequences for mempool design, and for other EIPs such as inclusion lists. However, because the accounts are listed statically in the outer transaction it is possible to modify transaction propagation rules so that conflicting transactions are not forwarded.
 
 ## Security Considerations
 
-Many security considerations with EIP-3074 are shared. Particularly, user wallets need to be very careful about which `contract_code` they sign.
+### Secure delegation
+
+The following is a non-exhaustive list of checks/pitfalls/conditions that delegate contracts *should* be wary of and require a signature over from the account's authority:
+
+ * Replay protection -- (ex. a nonce) should be implemented by the delegate and signed over. Without it, a malicious actor can reuse a signature, repeating its effects.
+ * `value` -- without it, a malicious sponsor could cause unexpected effects in the callee.
+ * `gas` -- without it, a malicious sponsor could cause the callee to run out of gas and fail, griefing the sponsee.
+ * `target` / `calldata` -- without them, a malicious actor may call arbitrary functions in arbitrary contracts.
+
+A poorly implemented delegate can *allow a malicious actor to take near complete control over a signer's EOA*.
+
+### Setting code as `tx.origin`
```

Comment:
What does that mean? Setting the code pointer address as tx.origin?

Reply:
Yes. I adjusted the title to be a bit clearer.